flow/src/main/java/com/sequenceiq/flow/service/FlowService.java

```diff
@@ -139,6 +139,7 @@ public FlowCheckResponse getFlowChainState(String chainId) {
             flowCheckResponse.setHasActiveFlow(!completed("Flow chain", chainId, relatedChains, relatedFlowLogs));
             flowCheckResponse.setLatestFlowFinalizedAndFailed(isFlowInFailedState(relatedFlowLogs, failHandledEvents));
             setEndTimeOnFlowCheckResponse(flowCheckResponse, relatedFlowLogs);
+            setStateEventFlowTypeOnFlowCheckResponse(flowCheckResponse, relatedFlowLogs,"flow chain", chainId);
```

Comment:
Missing space: relatedFlowLogs,"flow chain"

Reply:
@lturcsanyi Updated that. TYSM!!